ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -243,44 +245,42 @@ void creatingSignerDisplaysFailureToCreateSignerText() {
 
   @Test
   void settingTlsKnownClientAndDisablingClientAuthenticationShowsError() {
-    String cmdLine = validBaseCommandOptions();
-    cmdLine += "--tls-allow-any-client ";
-    final boolean result =
-        parser.parseCommandLine((cmdLine + subCommand.getCommandName()).split(" "));
-
+    List<String> cmdLine = validBaseCommandOptions();
+    cmdLine.add("--tls-allow-any-client");
+    final boolean result = parser.parseCommandLine(cmdLine.toArray(String[]::new));
     assertThat(result).isFalse();
     assertThat(commandError.toString()).contains("expected only one match but got");
   }
 
   @Test
   void tlsClientAuthenticationCanBeDisabledByRemovingKnownClientsAndSettingOption() {
-    String cmdLine = validBaseCommandOptions();
-    cmdLine = removeFieldFrom(cmdLine, "tls-known-clients-file", "tls-allow-ca-clients");
-    cmdLine += "--tls-allow-any-client ";
-    final boolean result =
-        parser.parseCommandLine((cmdLine + subCommand.getCommandName()).split(" "));
+    List<String> cmdLine = validBaseCommandOptions();
```

Comment:
nit: final

Reply:
can't - as it gets reset in the following line.